OMEdit/OMEditLIB/Simulation/SimulationOutputWidget.cpp

```diff
@@ -982,12 +1049,9 @@ void SimulationOutputWidget::compilationProcessFinished(int exitCode, QProcess::
     exitCodeStr.append("\nTry compiling with the default MinGW compiler. Select \"MinGW\" in \"Tools->Options->Simulation->Target Build\".");
   }
   if (exitStatus == QProcess::NormalExit && exitCode == 0) {
-    writeCompilationOutput(tr("Compilation process finished successfully."), Qt::blue);
+    writeCompilationOutput(tr("Compilation process finished successfully.")+"\n", Qt::blue);
```

Comment:
Same as above.

Reply:
yes, but that avoids having to re-write the translation